code/website/index.html

```diff
@@ -1,22 +1,47 @@
 <h2>Drasil - Generate All the Things!</h2>
+<p id="center"><a href="https://github.com/JacquesCarette/Drasil">Link to GitHub repository</a></p>
+<hr>
+<br />
+<p>
+  This webpage is designed to contain the most up to date
+  <a href="#caseStudies">case study artifacts</a>
+  and <a href="#packages">package dependency graphs</a> from the Drasil repository.
+  The case study artifacts include the Software Requirements Specification (SRS) for the case study,
+  which specifies what the program sets out to achieve.
+  The package dependency graphs shows the hierarchy of modules within each package.
+  The footer of this page contains the continuous integration build of the project,
+  as well as the commit number that the build and artifacts are based off of.
+</p>
+<br />
+<a name="caseStudies"></a>
+<h3>Case Studies</h3>
+<p>Each of the case studies contain their own generated PDF and HTML reports,
+and in some cases, their own generated code.</p>
 <ul>
 $-for(examples)$
-  <li>$name$ Artifacts<br>
+  <li><b>$name$</b> - 
+    $desc$
+    <br />
+    <br />
     <ul>
-    $-for(srs)$
-      <li><a href="$url$">$name$ SRS [$type$]</a></li>$sep-$
-    $endfor$
+      <li>$name$ SRS
+        $-for(srs)$
+          <a href="$url$">[$type$]</a>
+        $endfor$
+      </li>
     $-if(src)$
-      <li><a href="$src-$">Generated Code</a></li>
```

Comment:
Any particular reason the `-` was removed here? Does it not affect the generated file's spacing?

Reply:
Does not seem to make a difference on my browser.